src/components/forms/AddPetForm.tsx

```diff
@@ -0,0 +1,206 @@
+          )}
+        />
+        <FormField
+          control={form.control}
+          name="species"
+          render={() => (
+            <FormItem>
+              <FormLabel>Species*</FormLabel>
+              <Select>
+                <SelectTrigger>
+                  <SelectValue placeholder="Select species" />
+                </SelectTrigger>
+                <SelectContent className="bg-white">
+                  <SelectItem value="Cat">Cat</SelectItem>
+                  <SelectItem value="Dog">Dog</SelectItem>
+                  <SelectItem value="Other">Other...</SelectItem>
+                </SelectContent>
+              </Select>
+            </FormItem>
+          )}
+        />
+        <FormField
+          control={form.control}
+          name="sex"
+          render={() => (
+            <FormItem>
+              <FormLabel>Sex</FormLabel>
+              <FormControl>
+                <Select>
+                  <SelectTrigger>
+                    <SelectValue placeholder="Select" />
+                  </SelectTrigger>
+                  <SelectContent className="bg-white">
+                    <SelectItem value="male">Male</SelectItem>
+                    <SelectItem value="female">Female</SelectItem>
+                  </SelectContent>
+                </Select>
+              </FormControl>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+        <FormField
+          control={form.control}
+          name="neutered"
+          render={() => (
+            <FormItem>
+              <FormLabel>Neutered</FormLabel>
+              <FormControl>
+                <Select>
+                  <SelectTrigger>
+                    <SelectValue placeholder="Select" />
+                  </SelectTrigger>
+                  <SelectContent className="bg-white">
+                    <SelectItem value="yes">Yes</SelectItem>
+                    <SelectItem value="no">No</SelectItem>
+                  </SelectContent>
+                </Select>
+              </FormControl>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+        <FormField
+          control={form.control}
+          name="dateOfBirth"
+          render={({ field }) => (
+            <FormItem>
+              <FormLabel>Date of Birth</FormLabel>
+              <FormControl>
+                <Input type="text" placeholder="YYYY-MM-DD" {...field} />
+              </FormControl>
+              <FormMessage className="text-danger" />
+            </FormItem>
+          )}
+        />
+        <FormField
+          control={form.control}
+          name="breed"
+          render={({ field }) => (
+            <FormItem>
+              <FormLabel>Breed</FormLabel>
+              <FormControl>
+                <Input placeholder="European" {...field} />
+              </FormControl>
+              <FormMessage className="text-danger" />
+            </FormItem>
+          )}
+        />
```

Comment:
I'm thinking about shortening this file, because its super long. 
What would you say to take FormField as separate component, and only pass props such as:
- name
-  label
- placeholder
- any other needed

If we display all info that we will need, then file would get even much longer than it is now, (name, breed, weight, size, personality, tags, etc.) so we can shorten it by creating reusable inputs
 <TextFormInut name="breed" label="Breed placeholder="European" />

Ideally, I would also consider moving FormSchema into a separate file, but I am not entirely sure if it would improve readability.

Reply:
Good idea, I'll try to implement it. Not sure how this will work for the date picker component though.
(at the moment date input is a regular text input, but I plan to rework it into a date picker).